libres/tests/analysis/test_update.cpp

```diff
@@ -0,0 +1,203 @@
+#include <string>
+#include <unordered_map>
+#include <vector>
+#include <cmath>
+#include <algorithm>
+
+#include <catch2/catch.hpp>
+
+#include <ert/util/rng.h>
+#include <ert/enkf/enkf_util.hpp>
+#include <ert/res_util/matrix_blas.hpp>
+
+#include <ert/analysis/update.hpp>
+#include <ert/analysis/std_enkf.hpp>
+
+namespace analysis {
+void run_analysis_update_without_rowscaling(analysis_module_type *module,
+                                            const bool_vector_type *ens_mask,
+                                            const meas_data_type *forecast,
+                                            obs_data_type *obs_data,
+                                            rng_type *shared_rng,
+                                            matrix_type *E, matrix_type *A);
+}
+
+const double a_true = 1.0;
+const double b_true = 5.0;
+
+struct model {
+    double a;
+    double b;
+
+    model(double a, double b) : a(a), b(b) {}
+
+    model(rng_type *rng) {
+        double a_std = 2.0;
+        double b_std = 2.0;
+        // Priors with bias
+        this->a = enkf_util_rand_normal(a_true + 0.5 * a_std, a_std, rng);
```

Comment:
You could make names for the bias paramaters as well - e.g. `a_bias = 0.5;` and `b_bias = 0.5;`

Reply:
More explicit - I like it and agree.

Reply:
With a tiny modification: Suggest `a_bias = 0.5 * a_std` and similar for `b`, as this is the bias in statistical terminology at least.